modules/social_features/social_swiftmail/social_swiftmail.module

```diff
@@ -54,6 +55,21 @@ function social_swiftmail_preprocess_swiftmailer(array &$variables) {
     $variables['site_slogan'] = '"' . t('Make your people bloom') . '"';
   }
 
+  // Add variables header and footer if set.
+  if ($social_swiftmail_config->get('template_header')['value']) {
+    // Sanitizing variables first to filter out Xss.
+    $template_header = Xss::filter($social_swiftmail_config
+      ->get('template_header')['value'], ['a', 'em', 'strong', 'u', 'br']);
+    // Create markup so it will be converted to html in the email.
+    $variables['header'] = Markup::create($template_header);
```

Comment:
We should replace most of this with:
```suggestion
    $template_header = $social_swiftmail_config->get('template_header');
    $variables['header'] = [
      '#type' => 'processed_text',
      '#text' => $template_header['value'],
      '#format' => $template_header['format'],
    ];
```

Although I realise that this would allow administrators with the `full_html` format to enter any arbitrary HTML, I'll argue that this is desired behaviour. When we trust a user on a website with this text_format then we should not limit what they can do (but merely instruct that something is a bad idea). If you agree then we may be able to remove the mail_html format altogether for any sites that do want to force a certain text format beyond what the distribution offers this can easily be done with an alter hook. If you disagree then we should force the use of html_mail here but that may cause some distribution bug reports.

Reply:
Found in https://drupal.stackexchange.com/a/235028/2166 👍